src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -119,8 +120,9 @@ export default function LineNumberTooltip({
     }
     setTimeout(() => {
       if (lineNumber === lastHoveredLineNumber.current) {
-        if (features.codeHeatMaps) {
-          dispatch(setBreakpointHitCounts(source.id));
+        if (codeHeatMaps) {
+          dispatch(setBreakpointHitCounts(source.id, () => setCodeHeatMaps(false)));
```

Comment:
we could do `dispatch(setBreakpointHitCounts(source.id).catch` here

Reply:
in my experience something closer to the protocol was already catching, so the catch here was not getting hit